drivers/network/ndis/include/miniport.h

```diff
@@ -96,6 +96,8 @@ typedef struct _LOGICAL_ADAPTER
     HARDWARE_ADDRESS            Address;                /* Hardware address of adapter */
     ULONG                       AddressLength;          /* Length of hardware address */
     PMINIPORT_BUGCHECK_CONTEXT  BugcheckContext;        /* Adapter's shutdown handler */
+    PBUS_INTERFACE_STANDARD     BusInterface;
+    BOOLEAN                     BusInterfaceQueried;
```

Comment:
```suggestion

```
 You can just check the GetBusData or SetBusData callback for NULL instead of having a variable for them.